packages/mobile/src/features/videoPlayer/commponents/YouTubePlayer.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Platform } from "react-native";
+import YouTube from "react-native-youtube";
+import { IPlayer } from "../models/IPlayer";
+import Global from '../../common/models/Global';
+
+
+export const YouTubePlayer = (props:IPlayer) =>{
+
+    const height = (Platform.OS=='ios')?"100%":0;
+
+    console.log(Global.env.GOOGLE_KEY,props);
+
+    return (
+      <YouTube
+        //@ts-ignore-start
+        apiKey={Global.env.GOOGLE_KEY}
```

Comment:
nie da się to inaczej ograć niż ts-ignore, próbowałeś as any

Reply:
Niestety any nie pomoże bo nie chodzi o to że jest zły typ propsa a o to że wedlug typów on nie potrzebuje tego propsa (a bez niego na iOS nie działa)